### DISCUSSION TITLE
Add huggingface profile link

### DISCUSSION THREAD

**Discussion by suwesh** [academicpages/academicpages.github.io#2348] [Q&A]
Hi, how to add huggingface profile link in the _config.yaml file to appear in the left side links sections like github and kaggle.

**Comment by suwesh** [comment]
![image](https://github.com/user-attachments/assets/b12e5625-e0d4-4353-b120-741302734e74)

      
 found a solution by editing includes/author-profile.html but huggingface icon is not appearing

**Comment by rjzupkoii** [comment]
Looks like Font Awesome has the [Hugging Face icon](https://fontawesome.com/v6/search?q=hugging%20face&o=r) listed under their Pro Tier icons. Unfortunately, that means that we really aren't in a position to add it to the template at this time, but we can give you some support on the process for upgrading Font Awesome within the template if you get the Pro Tier icons.

**Comment by suwesh** [comment]
Thanks but ig i found a work around by just typing the emoji lol
![image](https://github.com/user-attachments/assets/4a8195dc-ba4e-4c07-894a-a953fb35be24)
